spec/CookiePluginSpec.php

```diff
@@ -146,7 +146,12 @@ function it_saves_cookie(RequestInterface $request, ResponseInterface $response,
 
         $response->hasHeader('Set-Cookie')->willReturn(true);
         $response->getHeader('Set-Cookie')->willReturn([
-            'cookie=value',
+            sprintf(
+                'cookie=value; expires=%s; Max-Age=60; path=/; domain=test.com; secure; HttpOnly',
+                (new \DateTime())->add(
+                    new \DateInterval('PT60S')
```

Comment:
There are no additional checks on the expires DateTime value in the Cookie object. Therefor not time critical. We could put a fixed cookie date string.